cli/src/commands/jobs.rs

```diff
@@ -193,6 +193,12 @@ pub async fn handle_submission(api: &mut PhylumApi, matches: &clap::ArgMatches)
         unreachable!();
     }
 
+    // Avoid request error without dependencies.
+    if packages.len() == 0 {
```

Comment:
```suggestion
    if packages.is_empty() {
```
Clippy is failing in CI, with this suggested change.